### ISSUE TITLE
Explorer & Manager: display verified badges if project has verified project socials

### ISSUE THREAD

**Issue by michellema1208** [gitcoinco/grants-stack#1551]
**User Story:**
As a project owner 
I want to display my verified badges on my project's Explorer page 
So that donors can see that my project socials have been verified 

**Acceptance Criteria**

**_Explorer_**
GIVEN that I have a verified Twitter and/or Github 
WHEN I look at my project's Explorer page 
THEN I see verified badges next to the verified socials 

**_Manager_**
Given that I have a verified Twitter and/or Github 
WHEN a Program Manager reviews my application on Manager 
THEN they see verified badges next to the verified socials 

**Product & Design Links:**
- [Explorer](https://www.figma.com/file/cgsZMTG71f2Hf89J38simK/Explorer-S17?node-id=526-14125&t=X3OZ4hS646JqLl2d-0)
- Manager: https://www.figma.com/file/V113F2Pp5m2qDM5odXjfFu/Manager-S17?node-id=1638-42123&t=ztT1gf2oxzz0ZBKa-4

**Tech Details:**

- [ ] Make sure that the verified badges on all Builder & Manager & Explorer are consistent with Figma (see above). Currently, the Builder verified badge looks different from the Figma and needs to be updated

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

**Comment by michellema1208** [comment]
@melissa-neira @willsputra this should be XSmall (hopefully), but can one of you iterate on verified badges next to socials on a project's Explorer page? Or link existing designs in this ticket? 

Noticing that we don't have that currently in Explorer - may have been a gap from before the two teams merged!

**Comment by melissa-neira** [comment]
@michellema1208 there's a design for that already [here](https://www.figma.com/file/cgsZMTG71f2Hf89J38simK/Explorer-S17?node-id=526-15098&t=juECbmTeFe9Bpsgw-4)

**Comment by michellema1208** [comment]
oh amazing, thank you @melissa-neira ! Three follow up qqs for you and @willsputra : 

1. Is there a design with verified badges for socials on Manager? For reviewing applications (also noticed this is missing) 
2. I noticed that our badge designs are slight different on Builder vs. Explorer: 

Builder (from app currently): 
<img width="1419" alt="Screen Shot 2023-04-21 at 12 27 48 PM" src="https://user-images.githubusercontent.com/26017180/233688150-68a23318-a6b0-46f1-b346-525263fcd191.png">

Explorer (from Figma): 
<img width="900" alt="Screen Shot 2023-04-21 at 12 28 04 PM" src="https://user-images.githubusercontent.com/26017180/233688131-f2d33b3f-e81c-4a70-bd4e-7a0ef895bfcc.png">

3. Thoughts on if it would be good to have a hover over the verified badge to explain what verified means? I think this is low priority but would love design input on necessity / I can add a ticket to backlog

**Comment by willsputra** [comment]
> I noticed that our badge designs are slight different on Builder vs. Explorer. Which one should we keep?

let's keep the one on Explorer! I used the same label design on grants hub figma too actually. the filled green shield is definitely more noticeable.
<img width="564" alt="image" src="https://user-images.githubusercontent.com/19514207/233690844-a49c99b2-fee2-4b84-873c-81dedc74b466.png">

> Thoughts on if it would be good to have a hover over the verified badge to explain what verified means? I think this is low priority but would love design input on necessity / I can add a ticket to backlog

yeah I think this is a great idea!

**Comment by michellema1208** [comment]
@willsputra amazing, I'll make that a task in the ticket to make all the badges consistent with Figma. 

Here's a separate ticket for the hover/info tooltip: https://github.com/gitcoinco/grants-stack/issues/1556. We can review this at a future Backlog Squad meeting!

**Comment by michellema1208** [comment]
@melissa-neira no rush, but could you link me to a Figma screen where it shows the verified badges on the individual applications in Manager? I'm not sure which ones in Figma are the most up-to-date

**Comment by melissa-neira** [comment]
@michellema1208 [here](https://www.figma.com/file/V113F2Pp5m2qDM5odXjfFu/Manager-S17?node-id=1638-42123&t=ztT1gf2oxzz0ZBKa-4) you go! 💗

**Comment by KurtMerbeth** [comment]
Blocked by #1528 
SInce this issue includes to make sure that the verified badges on all Builder & Manager & Explorer are consistent,
we should merge 1528 first to have a clean base and avoid merge conflicts

**Comment by michellema1208** [comment]
closing this ticket in favor of new ticket: https://github.com/gitcoinco/grants-stack/issues/2344
